server.go

```diff
@@ -4,7 +4,7 @@ import (
 	"log"
 	"net/http"
 	"os"
-	"vsc-node/graph"
```

Comment:
This file can be removed as we won't be using it.

Reply:
Deleted the server.go